src/main/java/fr/esgi/al5/tayarim/services/ReservationService.java

```diff
@@ -147,7 +148,13 @@ public ReservationDto createReservation(@NonNull ReservationCreationDto reservat
 
     String status;
     if (isAdmin) {
-      status = "payed";
+      if(dateDepart.isBefore(LocalDate.now()) || dateDepart.isEqual(LocalDate.now())){
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.